lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.cpp

```diff
@@ -434,6 +434,33 @@ GetNominal(swift::Demangle::Demangler &dem, swift::Demangle::NodePointer node) {
   return {};
 }
 
+/// Detect the AnyObject type alias.
+static bool IsAnyObjectTypeAlias(swift::Demangle::NodePointer node) {
+  using namespace swift::Demangle;
+  if (!node || node->getKind() != Node::Kind::TypeAlias)
+    return false;
+  if (node->getNumChildren() < 2)
+    return false;
+  NodePointer module = node->getChild(0);
+  if (!module || !module->hasText() || module->getText() != swift::STDLIB_NAME)
+    return false;
+  NodePointer ident = node->getChild(1);
+  if (!ident || !ident->hasText() || ident->getText() != "AnyObject")
```

Comment:
Nit, you could replace these two lines with:
```
return ident && ident->hasText() && ident->getText() == "AnyObject";
```

Reply:
I didn't want to make it harder to read by suddenly inverting the conditions on the last line.